data/tokens.json

```diff
@@ -1519,9 +1535,9 @@
             "description": "The primary color used for interactive elements."
           },
           "hover": {
-            "value": "{color.palette.darkblue.600}",
+            "value": "{color.text.interactive.default}",
             "type": "color",
-            "description": "The hover color used for interactive elements."
+            "description": "The hover color used for interactive text elements, like links."
```

Comment:
This change appears to be following the old instructions instead of the new ones. Light theme should reflect the same changes as the Dark theme tokens. (If I'm reading the code correctly)